content/docs/user-guide/related-technologies.md

```diff
@@ -0,0 +1,135 @@
+# Comparison with Related Technologies
+
+DVC combines a number of existing ideas into a single tool, with the goal of
+bringing best practices from software engineering into the data science field
+(refer to [What is DVC?](/doc/user-guide/what-is-dvc) for more details).
+
+### Git
```

Comment:
`###` -> `##` ?

Reply:
H2 looks too big and produces entries in the right sidebar. We also skip to H3s directly in the Basic Concepts guide, BTW.

Reply:
but to some extent that's the whole point - to have a navigation?

Reply:
You're right, updated in 98ffea36.